src/features/ticket/api/ticketOption.ts

```
@@ -0,0 +1,68 @@
+// 티켓 옵션 수정
+export const modifyTicketOption = async (
+  ticketOptionId: number,
+  data: TicketOptionRequest
+): Promise<TicketOptionRequest> => {
+  const response = await axiosClient.put<TicketOptionRequest>(`/ticket-options/${ticketOptionId}`, data);
+  return response.data;
+};
+
+// 티켓 옵션 삭제 (UI 없음, 현재 호출되지 않고있는 상태)
+export const deleteTicketOption = async (ticketOptionId: number): Promise<ApiResponse<{}>> => {
+  const response = await axiosClient.delete<ApiResponse<{}>>(`/ticket-options/${ticketOptionId}`);
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)